packages/playground-ui/src/services/mastra-runtime-provider.tsx

```diff
@@ -97,7 +97,15 @@ export function MastraRuntimeProvider({
         topK,
         topP,
         instructions,
-        ...(memory ? { threadId, resourceId: agentId } : {}),
+        ...(memory ? { 
+          threadId, 
+          resourceId: agentId,
+          memoryOptions: {
+            threads: {
+              generateTitle: true,
```

Comment:
so playground still generates titles

Reply:
[extra] just a suggestion: exposing the _generateTitle_ **prompt** could help with instruction changes like translation. Let me know if you prefer this as an issue.

Reply:
@andersonba that's a great idea! opening an issue is definitely the best way to track it